src/wstool/cli_common.py

```diff
@@ -162,12 +162,18 @@ def _get_status_flags(basepath, elt_dict):
          elt_dict['actualversion'] is not None and
          elt_dict['specversion'] != elt_dict['actualversion'])):
         mflag += 'V'
-    if ('remote_revision' in elt_dict and
-        elt_dict['remote_revision'] != '' and
-        elt_dict['remote_revision'] is not None and
-        elt_dict['actualversion'] is not None and
-        'actualversion' in elt_dict and
-        elt_dict['remote_revision'] != elt_dict['actualversion']):
+    if (('remote_revision' in elt_dict and
+         elt_dict['remote_revision'] != '' and
+         elt_dict['remote_revision'] is not None and
+         elt_dict['actualversion'] is not None and
+         'actualversion' in elt_dict and
```

Comment:
This probably isn't related to this pr, but the check for `'actualversion' in elt_dict` should probably come before the check `elt_dict['actualversion'] is not None`, in order to avoid a `KeyError`.